general/open-source.md

```diff
@@ -0,0 +1,18 @@
+# 📖 Langley Foxall Open Source
+We've recently began delving into the open source world as a company, by opening up some of the tools developed for
+internal purposes to the open source community.
+
+We love contributions and want our packages to help people, so if you have suggestions for improvements, please file
+an issue on the relevant repository.
+
+##### Laravel
+* [Xero - An Eloquent-esque interface for the Xero PHP Wrapper](https://github.com/langleyfoxall/xero-laravel)
+* [Modules - A package for building modular laravel applications](https://github.com/langleyfoxall/modules-laravel)
+* [Helpers - A useful set of helper functions for use in Laravel](https://github.com/langleyfoxall/helpers-laravel)
+
```

Comment:
Laravel Boilerplate is missing.

Reply:
I never referred to the boilerplate as open source - as we've written nothing and just dragged in a few packages to a Laravel project - if people want to see it included though crack on.

Reply:
But Laravel is built off the back of many libraries, and still classed as open-source. I think the boilerplate should totally be added.